docs/SDL/OnAppPermissionChanged/index.md

```diff
@@ -22,7 +22,9 @@ Purpose
 |appPermissionsConsentNeeded|Boolean|false||
 |appUnauthorized|Boolean|false||
 |priority|[Common.AppPriority](../../common/enums/#apppriority)|false||
-|requestType|[Common.RequestType](../../common/enums/#requesttype)|false|array: true<br>minsize: 0<br>maxsize: 100|
+|requestType|[Common.RequestType](../../common/enums/#requesttype)|false|array: true<br>minsize: 0<br>maxsize: 100|  
+|requestSubType|String|false|array: true<br> minsize: 0<br> maxsize: 100 <br> maxlength: 100|The list of SystemRequest's requestSubTypes allowed by policies for the named application.<br>If the app sends a requestSubType which is not specified in this list, then that request should be rejected.<br>An empty array signifies that any value of requestSubType is allowed for this app.<br> If this parameter is omitted, then a request with any value of requestSubType is now allowed for this app|
```

Comment:
There doesn't appear to be a 5th column in this table, so the description is ignored in this case

Reply:
fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/98/commits/b882061b9308da7dbc4d0c60778d91b558bfb46c